### PR TITLE
refactor: Refactored Unit Tests for `Geometry` models in swift.

### PR DESCRIPTION
**Pull request by iHackSubhodip**
List of Refactored Unit Test files, Adding Obj-C uni tests file for reference:

1. [GMUGroundOverlayTest.m](https://github.com/googlemaps/google-maps-ios-utils/blob/main/Tests/GoogleMapsUtilsObjCTests/unit/Geometry/GMUGroundOverlayTest.m)
2. [GMULineStringTest.m](https://github.com/googlemaps/google-maps-ios-utils/blob/main/Tests/GoogleMapsUtilsObjCTests/unit/Geometry/GMULineStringTest.m)
3. [GMUPointTest.m](https://github.com/googlemaps/google-maps-ios-utils/blob/main/Tests/GoogleMapsUtilsObjCTests/unit/Geometry/GMUPointTest.m)
4. [GMUPolygonTest.m](https://github.com/googlemaps/google-maps-ios-utils/blob/main/Tests/GoogleMapsUtilsObjCTests/unit/Geometry/GMUPolygonTest.m)
5. [GMUStyleTest.mm](https://github.com/googlemaps/google-maps-ios-utils/blob/main/Tests/GoogleMapsUtilsObjCTests/unit/Geometry/GMUStyleTest.mm)

Refactor:

1. Refactored Unit Tests for `Geometry` models in swift.
2. Corrected the class names in code comments and properties, which got missed in last PR.

### CODE REVIEW

File: Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPolygonTest1.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+@testable import GoogleMapsUtils
+
```
Comment (wangela) [inline Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPolygonTest1.swift:18]: Doesn't this require an import of GoogleMaps to access GMSMutablePath?

Comment (wangela) [reply Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPolygonTest1.swift:18]: And CoreLocatin to get CLLocation*?

File: Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPointTest1.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+@testable import GoogleMapsUtils
+
```
Comment (wangela) [inline Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPointTest1.swift:18]: Do we need to import CoreLocation?

**Review by wangela — CHANGES_REQUESTED** [review]
Since the new Swift tests don't collide with any existing tests, let's give them their final filenames (without the "1" suffix), e.g. `Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUGroundOverlayTest.swift` and remove the rename TODOs on L19 of those files.

File: Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPolygonTest1.swift (hunk 1 above)
Comment (iHackSubhodip) [reply Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUPolygonTest1.swift:18]: In my local tests are passing, using Xcode 15.4, do you want me to add those import, for a safer side?

**Comment by iHackSubhodip** [comment]
> Since the new Swift tests don't collide with any existing tests, let's give them their final filenames (without the "1" suffix), e.g. `Tests/GoogleMapsUtilsSwiftTests/unit/Geometry/Models/GMUGroundOverlayTest.swift` and remove the rename TODOs on L19 of those files.

Aah, yes! Nice catch, pushed the changes by renaming the files. Thanks for the quick review.

**Review by wangela — APPROVED** [review]

**Pull request merged**